codeclimate.gemspec

```
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = "~> 2.5"
 
-  s.add_dependency "activesupport", ">= 5.1.3", "< 5.2"
+  s.add_dependency "activesupport", ">= 5.2.4.3", ">= 5.2"
```

Can this be simplified?
```suggestion
  s.add_dependency "activesupport", ">= 5.2"
```
Do we have any comments about why it was previously explicitly keeping it < 5.2?

---

yeap, done ! 👍🏼